api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -110,7 +110,9 @@ message AttributeContext {
     // The HTTP request `Host` or 'Authority` header value.
     string host = 5;
 
-    // The HTTP URL scheme, such as `http` and `https`.
+    // The HTTP URL scheme, such as `http` and `https`. This is set for HTTP/2
+    // requests only. For HTTP/1.1, use "x-forwarded-for" header value to lookup
+    // the scheme of the request.
```

Comment:
cc @alyssawilk you might want to unwind this as part of https://github.com/envoyproxy/envoy/pull/14899?

Reply:
Unfortunately this comment is misleading as-is.  It's correct for L1s, but for L2 HTTP/2 scheme does not reflect the URL scheme of the client request, and XFP is the correct thing to match on for both HTTP/1 and HTTP/2.

With my PR in flight, :scheme will always be set, but will still not be the correct thing to check at L1 Envoys.  With the next PR, :scheme will always be set and will be the source of truth.  I expect that to land in ~2 weeks, as I'd like some time between step 1) and 2) just to lower risk.